generic_petg.xml.fdm_material

```diff
@@ -64,6 +64,28 @@ Generic PETG profile. The data in this file may not be correct for your specific
             </hotend>
         </machine>
 
+        <machine>
+            <machine_identifier manufacturer="Ultimaker B.V." product="Ultimaker 3"/>
+            <machine_identifier manufacturer="Ultimaker B.V." product="Ultimaker 3 Extended"/>
+            <setting key="heated bed temperature">85</setting>
+            <hotend id="BB 0.4" />
+            <hotend id="BB 0.8" />
+            <hotend id="AA 0.25">
+                <setting key="hardware compatible">yes</setting>
+                <setting key="print temperature">230</setting>
+                <setting key="standby temperature">100</setting>
```

Comment:
Shouldn't we add this value to the 0.8 nozzle as well? If yes, then this value is equal for all nozzles and can be moved to line #70 to avoid duplications.

Reply:
I did add it to 0.8 nozzle